src/Data/PQueue/Prio/Min.hs

```diff
@@ -142,13 +142,6 @@ import qualified Data.PQueue.Prio.Internals as Internals
 
 import Prelude hiding (map, filter, break, span, takeWhile, dropWhile, splitAt, take, drop, (!!), null)
 
-#ifdef __GLASGOW_HASKELL__
-import GHC.Exts (build)
-#else
-build :: ((a -> [a] -> [a]) -> [a] -> [a]) -> [a]
-build f = f (:) []
-#endif
-
```

Comment:
Is this not used anymore or is it just to remove non-GHC stuff?

Reply:
It's not used in this module anymore. We still do it elsewhere, and this PR is not the place to deal with that.